src/main/rules/GCI1066/GCI1066.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Logging Format Interpolation - Avoid using immediate string formatter in logging calls.",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "creedengo",
```

Comment:
Maybe add python tag as it seems only specific to python

Reply:
I'm not ok to add "python" tag ... if this rule is python specific, then there will be only an implementation in python plugin and not inside other plugins.